lib/views/pages/profile/profile_page.dart

```diff
@@ -1,79 +1,108 @@
 import "package:flutter/material.dart";
 import "package:hooks_riverpod/hooks_riverpod.dart";
 import "package:proxima/viewmodels/profile_view_model.dart";
-import "package:proxima/views/pages/profile/posts_info/info_card.dart";
+import "package:proxima/views/pages/profile/posts_info/info_card_badge.dart";
+import "package:proxima/views/pages/profile/posts_info/info_card_comment.dart";
+import "package:proxima/views/pages/profile/posts_info/info_card_post.dart";
 import "package:proxima/views/pages/profile/posts_info/info_column.dart";
 import "package:proxima/views/pages/profile/posts_info/info_row.dart";
-import "package:proxima/views/pages/profile/user_info/centauri_points.dart";
 import "package:proxima/views/pages/profile/user_info/user_account.dart";
 
 /// This widget is used to display the profile page
 /// It contains the user info, centauri points, badges, posts and comments
 class ProfilePage extends HookConsumerWidget {
   const ProfilePage({super.key});
+  static const postTabKey = Key("postTab");
+  static const commentTabKey = Key("commentTab");
+  static const tabKey = Key("tab");
+  static const postColumnKey = Key("postColumn");
+  static const commentColumnKey = Key("commentColumn");
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final asyncUserData = ref.watch(profileProvider);
 
-    final theme = Theme.of(context);
+    var itemListBadge = <InfoCardBadge>[];
+    var itemListPosts = <InfoCardPost>[];
+    var itemListComments = <InfoCardComment>[];
 
-    var itemList = <InfoCard>[];
+    BoxShadow shadow = BoxShadow(
+      color: Theme.of(context).colorScheme.shadow.withOpacity(0.4),
+      offset: const Offset(0, 1),
+      blurRadius: 0.1,
+      spreadRadius: 0.01,
+    );
 
-    for (var i = 0; i < 5; i++) {
-      itemList.add(
-        InfoCard(
-          theme: theme,
-        ),
+    //this is a MOCK list of cards
+    for (var i = 0; i < 10; i++) {
+      itemListBadge.add(
+        InfoCardBadge(shadow: shadow),
+      );
+
+      itemListPosts.add(
+        InfoCardPost(shadow: shadow),
+      );
+
+      itemListComments.add(
+        InfoCardComment(shadow: shadow),
       );
```

Comment:
Code to change if above suggestion are accepted:

```suggestion
      itemListPosts.add(
        InfoCardPost(
          shadow: shadow,
          title: "Post title",
          description:
              "My super post that talks about something that is super cool and is located in a super spot",
        ),
      );

      itemListComments.add(
        InfoCardComment(
          shadow: shadow,
          comment:
              "Here is a super comment on a super post that talks about something that is super cool and is located in a super spot that is very cosy and nice",
        ),
      );
```